### DISCUSSION TITLE
Trustmaster TCA Quadrant does not line up

### DISCUSSION THREAD

**Discussion by Gitty29** [flybywiresim/aircraft#2311] [Help]
I use the latest custom A32NX update

This are my settings in ThrottleConfiguration.ini:

[Throttle]
Log = true
Enabled = true
ReverseOnAxis = true
DetentReverseFull = -1.00
DetentIdle = 0.00
DetentClimb = 0.66
DetentFlexMct = 0.88
DetentTakeOffGoAround = 1.00

IDLE position
![Idle](https://user-images.githubusercontent.com/75734056/101630456-2db47680-3a23-11eb-90ff-03c8088841c6.png)

CLIMB position
![climb](https://user-images.githubusercontent.com/75734056/101630491-3ad16580-3a23-11eb-8cdf-c0623513ce6b.png)

FLEX position
![flex](https://user-images.githubusercontent.com/75734056/101630512-40c74680-3a23-11eb-8fbb-4a738099f7ac.png)

TOGA position
![toga](https://user-images.githubusercontent.com/75734056/101630534-46bd2780-3a23-11eb-9493-1420fed4c0b3.png)

What to do?

**Comment by pepperoni505** [comment]
Hello, would you mind please providing more information? The above pictures look like both throttles are lined up properly.

**Reply by Gitty29** [reply]
Hello,

No, they don't on these pictures.
When my thottle is in the CLIMB position you can see that the throttle in the simulator is in the FLEX position.
When my thottle is in the FLEX position you can see that the throttle in the simulator is between FLEX and TOGA.

I deleted my settings for the TCA throttle and now with the default settings (and default sensitivity) it looks like it's working.
Perhaps an idea to write this in the instructions.

Regards,
Robin

**Reply by pepperoni505** [reply]
Ah that's what you meant. I will let the team know about this issue. Thank you for bringing it up

**Reply by derl30n** [reply]
I've fixed this by editing the throttle axis curve on mine.

**Comment by aguther** [comment]
Hello,

I‘m sorry that you have issues. I assume you have assigned the throttles to MSFS directly and you are not running any software that changes the sensitivity of them.

Could you please post the log from the Developer Mode Console? When the console is open you can add „WASM“ in the filter, then go to full reverse. The values should show -1.0 as input. Then go to CL, FLX and TOGA and note the values that are reached then as input. Please post a screenshot if possible of that so we can learn.

Best,
Andreas

**Comment by aguther** [comment]
Ok just seen it’s working now. I will add a section that explains how to get the values.

The goal is of course to get any settings into an EFB or FMGC.

**Comment by derl30n** [comment]
This is my setting with no 3rd party software. Detents align and idle reverse is actually useable.

![image](https://user-images.githubusercontent.com/1857322/101660294-4c2c6900-3a47-11eb-894e-47fcd0889ab1.png)
